app/eventyay/plugins/sendmail/templatetags/sendmail_extras.py

```diff
@@ -0,0 +1,12 @@
+from django import template
+
+register = template.Library()
+
+@register.filter
+def split(value, delimiter=','):
```

Comment:
This is the disadvantage of using Django template instead of Jinja.
With Jinja, you don't need to define this filter. In Jinja template, you just use normal Python code:

```nunjucks
{{ values.split(',') }}
```